tutorials/abap-s4hanacloud-purchasereq-replace-wrapper/abap-s4hanacloud-purchasereq-replace-wrapper.md

```diff
@@ -0,0 +1,290 @@
+
+In the case of a released API, the authorization objects are released as well and so the mitigation is no longer necessary. In this case you can follow the standard procedure: you will first identify the necessary authorization objects, then you will maintain the corresponding authorization defaults.
+
+For simplicity and continuity, you will use the `Z_USER_###` user with the `ZR_SHOPCART_###` and the `ZAP_BC_ABAP_DEVELOPER_5_###` roles created in the previous tutorial [Provide Authorizations to Users for non-Released Authorization Objects checked by the "Create Purchase Requisition" function module](https://developers.sap.com/tutorials/abap-s4hanacloud-purchasereq-provide-authorization.html). Remove any other additional role from the `Z_USER_###` user and, in the `ZR_SHOPCART_###` role, make sure to select the Authorization Default for `ZUI_SHOPCART_WRAPPER_O4_###` (NOT the variant).
+
+1. The first step is to find out the needed authorization objects for the released API: logon to ADT and click on `Navigate`->`Open ABAP Development Object` and search for the purchase requisition behavior definition `I_PURCHASEREQUISITIONTP`. Open it. From the `Properties` tab navigate to `API State`: from here you can open the **Authorization Default Values** and get information on the needed authorization objects and relative field values (for more information check the Purchase Requisition API documentation). In this case, the needed authorization objects and field values are:
+
+    | Object       | Field Name     | Value
+| :-----       | :------------- | :-------------
+| `M_BANF_BSA` | `ACTVT`        | 01
+|              |                | 02
+|              |                | 03
+|              |                | 08
+|              | `BSART`        | *
+| `M_BANF_EKG` | `ACTVT`        | 01
+|              |                | 02
+|              |                | 03
+|              |                | 08
+|              | `EKGRP`        | `$EKGRP`
+| `M_BANF_EKO` | `ACTVT`        | 01
+|              |                | 02
+|              |                | 03
+|              |                | 08
+|              | `EKORG`        | `$EKORG`
+| `M_BANF_WRK` | `ACTVT`        | 01
+|              |                | 02
+|              |                | 03
+|              |                | 08
+|              | `WERKS`        | `$WERKS`
+
+2. Start transaction `SU22` and select `SAP Gateway OData V4 Backend Service Group and Assignments` from the dropdown menu of the `Type of Application` field. In the `Object Name` field input your Service Binding name (`ZUI_SHOPCART_WRAPPER_O4_###`) and click on the `Execute` button.
```

Comment:
Maybe add the change of environment, for clarity:
'Logon to the system via SAP GUI and start transaction 'SU22'...' etc